.circleci/wait-all-jobs-complete.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+function running_jobs_count() {
+    curl -s -u "4b586c69a308a01acfd126b225d2247cbe11c2d2:" \
```

Comment:
What means 4b586c69a308a01acfd126b225d2247cbe11c2d2?

Reply:
CircleCI API requires an access token to request info.
It is supposed to be CIRCLECI_TOKEN environment variable set in CircleCI config.

@edwarnicke can we have such a variable? The token from the code here is enough (or even prefered), since only reading capability is necessary.

Reply:
This is usually a bad practice. Even publishing it here sounds like a really bad security issue. Please remove.

Reply:
done